components/home/WorkoutContainer.jsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { View, StyleSheet } from 'react-native';
+import { Button, Subheading } from 'react-native-paper';
+
+const buttonStartLocation = 28;
+const textStartLocation = 30;
+
+export default (props) => (
+  <View>
+    {props.items.map((item, index) => (
+      <Button contentStyle={{ width: 100, height: 35 }} style={{ ...styles.startButton, top: buttonStartLocation + index * 70 }} color="#FFFFFF" onPress={() => alert(`This should start recording ${item.name}`)}>
+        Start
+      </Button>
+    ))}
+
+    {props.items.map((item, index) => (
+      <Subheading style={[{
+        position: 'absolute', top: textStartLocation + index * 70, left: 15, fontSize: 18,
+      }]}
+      >
+        {item.name}
+      </Subheading>
+    ))}
+  </View>
```

Comment:
You only need one map for this part, you can return both of the tags in the function.